sdks/python/apache_beam/yaml/examples/simple_filter.yaml

```diff
@@ -0,0 +1,41 @@
+# coding=utf-8
```

Comment:
are these yaml files packaged when releasing beam?

Reply:
I would assume everything under `apache_beam/` is but I suppose I am not 100% on that

Reply:
we could always create the temp csv file and run the example.

Reply:
Also fixed by moving csv to `apache-beam-testing`.